src/app/views/main/Bridge/Bridge.tsx

```
@@ -261,7 +261,13 @@ const BridgeView: React.FC<React.HTMLAttributes<HTMLDivElement>> = (props: any)
       providerOptions
     });
 
-    const provider = await web3Modal.connect();
+    const provider = await web3Modal.connect().catch(error => {
+      // handle user rejection
+      console.log(error)
```

We should handle the error properly instead of ignoring the error like this. Existing legitimate errors will be ignored if we catch and ignore.

---

Code changes have been made so this comment can be considered as acknowledged.